sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -402,191 +210,23 @@ For more information and samples using prebuilt models see:
 ### Train a Model
 Train a machine-learned model on your own form types. The resulting model will be able to recognize values from the types of forms it was trained on.
```

Comment:
We should be changing this line too

Reply:
I am not making modifications to the Readme in this PR. Will do in a future PR